trade-module/src/main/java/io/edpn/backend/trade/adapter/web/dto/filter/RestLocateCommodityFilterDto.java

```diff
@@ -12,7 +13,9 @@ public record RestLocateCommodityFilterDto(
         Boolean includePlanetary,
         Boolean includeOdyssey,
         Boolean includeFleetCarriers,
-        String maxLandingPadSize,
+        String shipSize,
         Long minSupply,
-        Long minDemand) implements LocateCommodityFilterDto {
+        Long minDemand,
+        RestPageFilterDto page
```

Comment:
This currently does not work. Im not sure if we can fix this with @Jacksonized. But with a corrected http request this throws mapping errors. Consider flattening and converting to a domain object via mapper, or investigate a way to make this work with a get request